src/main/java/racing/ResultView.java

```diff
@@ -13,17 +13,33 @@ private ResultView() {
 
     }
 
-    public static void printCars(List<Car> cars) {
-        cars.forEach(ResultView::printCar);
+    public static void printCars(Cars cars) {
+        cars.getCars().forEach(ResultView::printCar);
         System.out.println();
     }
 
     public static void printWinners(List<Car> cars) {
-        String names = cars.stream().map(Car::getName).collect(Collectors.joining(NAME_DELIMITER));
+        String names = cars.stream()
+                .map(Car::getName)
+                .map(CarName::toString)
+                .collect(Collectors.joining(NAME_DELIMITER));
         System.out.printf(WINNER_FORMAT, names);
     }
 
+    public static void printError(Exception exception) {
+        if (exception instanceof CarNameTooLongException) {
+            System.out.println("자동차 이름은 5자를 초과할 수 없습니다.");
+            return;
+        }
+
+        System.out.println("Unexpected error : " + exception);
+    }
```

Comment:
> else를 제거하긴 했는데 exception이 여러개 되면 굳이 중간에 중복되는 return이 들어가서 보기 좋진 않을 것 같아요.
오히려 핸들링 되지 않은 에러를 표현하기에 else만큼 적합한 것이 없을것 같다는 생각이 들었습니다.

예외 메시지를 예외 클래스 내부로 옮기고, `getMessage()`를 오버라이딩 해서 출력해보면 어떨까요?
```suggestion
    public static void printError(Exception exception) {
        System.out.println(exception.getMessage());
    }
```